src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -1243,19 +1254,39 @@ inline std::ostream& operator<<(std::ostream& stream,
   return stream;
 }
 
+/**
+ * @brief Ionospheric delay model configuration.
+ * @ingroup config_and_ctrl_messages
+ *
+ * @note
+ * If model is disabled (set to `OFF`), then that model will not be
+ * used in downstream calculations. If set to `AUTO`, the device will select
+ * the best available option.
+ */
+struct alignas(4) IonosphereConfig {
+  /** If not OFF -- the ionospheric delay model to be used. */
```

Comment:
Let's just say "The ionospheric delay model to use." Don't need the "it's not off" part since that's part of the enum definition.